.storybook/main.js

```diff
@@ -1,11 +1,30 @@
+const path = require("path");
+const toPath = (_path) => path.join(process.cwd(), _path);
+
 module.exports = {
-  "stories": [
+  stories: [
     "../src/**/*.stories.mdx",
     "../src/**/*.stories.@(js|jsx|ts|tsx)"
   ],
-  "addons": [
-    "@storybook/addon-links",
+  addons: [
+    "@storybook/addon-a11y",
     "@storybook/addon-essentials",
-    "@storybook/preset-create-react-app"
-  ]
+    "@storybook/addon-links",
+    "@storybook/addon-storysource",
+    "@storybook/preset-create-react-app",
+    "storybook-addon-performance/register"
+  ],
+  webpackFinal: async (config) => {
+    return {
+      ...config,
+      resolve: {
+        ...config.resolve,
+        alias: {
+          ...config.resolve.alias,
+          "@emotion/core": toPath("node_modules/@emotion/react"),
+          "emotion-theming": toPath("node_modules/@emotion/react")
```

Comment:
This is really funky, but I guess makes sense.

Reply:
Yeah, it's weird. I linked it in the description above, but found this solution from the [issue here](https://github.com/chakra-ui/chakra-ui/issues/2527#issuecomment-728161743).